src/server/user.ts

```diff
@@ -0,0 +1,22 @@
+export async function unsetFavoriteAd(adId: number): Promise<boolean> {
+    const answer = await fetch(`/user/favorite/${adId}`, {
+        method: 'DELETE'
+    }).then(response => {
+        // Need to think how to determine if the action succeeded
```

Comment:
I think that after delete/add we need to pull all the favorites again

Reply:
Done, also changed the userStore